test/common/config/subscription_factory_test.cc

```diff
@@ -91,6 +91,7 @@ TEST_F(SubscriptionFactoryTest, RestClusterSingleton) {
 
   config.mutable_api_config_source()->set_api_type(envoy::api::v2::core::ApiConfigSource::REST);
   config.mutable_api_config_source()->mutable_refresh_delay()->set_seconds(1);
+  config.mutable_api_config_source()->mutable_request_timeout()->set_seconds(1);
```

Comment:
Now that we have the prof code change locked down, can you audit all the test changes? In general I don't think you need to set the default all over the place. I would just make sure that you have a test that tests something other than the default in at least one place. The rest can probably be reverted.